tabs/other/secrets_validation/scripts/secrets_validation.py

```diff
@@ -79,6 +79,7 @@ def unzip_archive(archive_path, skip_parent_directory=True):
     elif inputs["provider"].lower() == "azure":
         validate_azure()
     elif inputs["provider"].lower() == "gcp":
-        validate_gcp()
+        #TODO
```

Comment:
Is there a task for it? if not please create a task and add here the number

Reply:
yes, I created here: https://cloudifysource.atlassian.net/jira/software/projects/RD/boards/56?assignee=5b9683120949fb340db41aa6&selectedIssue=RD-6950